web-console/4.0-designs/project-selector/project-selector.md

```diff
@@ -0,0 +1,16 @@
+# Project Selector: Favorites and Defaults
+
+The project selector will be accessible at all times from the secondary masthead.
+
+![image 1](img/1.png)
+![image 2](img/1.1.png)
+
+* The project selector dropdown will have two sections; favorites and defaults will be in the first section of the list and all other projects will be in the second section of the list. The sections will be divided by a horizontal separator.
+* Favorite projects
```

Comment:
Can we use the terminology "pinned projects" instead? Thinking the word "Favorites" may still seem too tied to the star icon, but really that is tied to the default project now and we are saying users can "pin" projects to the top of the list. Just an idea. It took me a minute to figure out what you meant.